include/kernels/dslash_wilson.cuh

```diff
@@ -98,7 +94,6 @@ namespace quda
 
           Link U = arg.U(d, gauge_idx, gauge_parity);
           HalfVector in = arg.in.Ghost(d, 1, ghost_idx + coord.s * arg.dc.ghostFaceCB[d], their_spinor_parity);
-          if (d == 3) in *= arg.t_proj_scale; // put this in the Ghost accessor and merge with any rescaling?
```

Comment:
again wondering for now.

Reply:
That’s gone because we now only support packing in the t dimension and not the copy directly from the t face.  That scale factor was only needed when copying, so without it, we simplify the code.

Reply:
Right, I remember now that you had mentioned that.